src/flat-config/language-options.d.ts

```diff
@@ -1,9 +1,5 @@
-import type {
-  EcmaVersion,
-  Parser,
-  ParserOptions,
-  SourceType,
-} from '../parser-options';
+import type { Linter } from 'eslint';
```

Comment:
We cant do this type import, because `@types/eslint` is only a devDependency https://github.com/Shinigami92/eslint-define-config/blob/629c46bd7b210a693e133281b190a1f0ddd663dc/package.json#L68

Because this project serves all its types without relying on external types, we need to define this manually

Reply:
I see

For the meantime, please use the following workaround:

```ts

type ParserModule =
        | {
              parse(text: string, options?: any): any;
          }
        | {
              parseForESLint(text: string, options?: any): any;
          };

// ...

parser?: Parser | ParserModule;
```

We still need the "old" behavior/Parser type, as we cannot rely that everyone out there has switched instantly to the newest eslint config.
This package sometimes make loose types like this so we do not annoy developers that are not that much up-to-date.

Reply:
fixed